governance-exercises/CivicSage Exercises/Ex01/2. forum.md

```diff
@@ -0,0 +1,10 @@
+## Draft any forum responses required or implied by your course of action.
+
+The Governance Facilitators propose structuring the changes to Spork listed by @Alice and @Faraz in three individual polls:
+1. Adjust Spork DAI Direct Deposit Module (D3M) Debt parameters.
+2. Increase Spork DAI deposit rate.
+3. Onboard ABC Always Blue Coin (ABC) token to Spork Protocol. 
+ 
```

Comment:
Good. Keep in mind that if the protocol is multichain, we should also specify which chain we are affecting. For instance:
"Spork Protocol Ethereum - Onboard ABC Always Blue Coin (ABC) token."